### PR TITLE
Study group 

### PR DESCRIPTION
**Pull request by truongduchuy**

### CODE REVIEW

**Comment by truongduchuy** [comment]
image

File: .eslintrc [hunk 1]
```diff
@@ -8,12 +8,14 @@
...
     "react/jsx-filename-extension": 0,
     //  proptypes is not required : 
     "react/prop-types": 0,
     "no-unused-vars":0,
-    "react/jsx-props-no-spreading":0
+    "react/jsx-props-no-spreading":0,
+    "endOfLine": 0,
+    "react/no-array-index-key": 0
   }
 }
```
Comment (fargusmd) [inline .eslintrc:21]: as we build in  a big team, please dont remove linters/or prettier configs ( to check betters diffs and kind of)

File: src/pages/Landing/components/Body/ChatBox/Input/Input.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import React from 'react'
+import happyIcon from 'assets/images/happy.png'
+import attachIcon from 'assets/images/attach.png'
+import microIcon from 'assets/images/micro.png'
+import styles from './Input.module.scss'
+
+export default () => (
+  <div className={styles.Input}>
+    <img src={happyIcon} alt="happy" />
+    <img src={attachIcon} alt="attach" />
```
Comment (fargusmd) [inline src/pages/Landing/components/Body/ChatBox/Input/Input.js:11]: would like to implement UI components on all form elements that project can have.
plus + using react-hook-form

File: src/pages/Landing/components/Body/ChatBox/Input/Input.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <input type="text" placeholder="Type messages here..." />
+    <img src={microIcon} alt="micro" />
+  </div>
```
Comment (fargusmd) [inline src/pages/Landing/components/Body/ChatBox/Input/Input.js:14]: reusable componnent

File: src/pages/Landing/components/Body/ChatBox/Input/Input.module.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    display: flex;
+    align-items: center;
+
+    img:first-child {
+        margin-right: .8rem;
+    }
+
+    img:nth-child(2) {
+        margin-right: 1.6rem;
+    }
```
Comment (fargusmd) [inline src/pages/Landing/components/Body/ChatBox/Input/Input.module.scss:14]: same. reusable comps

File: src/pages/Landing/components/Body/ChatBox/Message/Message.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+                <img src={fileIcon} alt="File" />
+              </div>
+              <div>
+                <div className={styles.FileName}>{file.fileName}</div>
+                <div className={styles.FileSize}>{file.fileSize}</div>
+              </div>
+            </div>
+            <img src={moreIcon} alt="more" />
+          </div>
+        )}
```
Comment (fargusmd) [inline src/pages/Landing/components/Body/ChatBox/Message/Message.js:31]: for Icons we have separate component wich transforms svg to React components
compoenents>Icons

File: src/pages/Landing/components/Body/FileBox/FileBox.js [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+    date: 'Oct 20, 2020 at 21:00',
+    size: '1.2MB',
+  },
+  {
+    isFile: false,
+    name: 'Muz.li',
+    date: 'Oct 20, 2020 at 21:00',
+    size: '1.2MB',
+  },
+]
```
Comment (fargusmd) [inline src/pages/Landing/components/Body/FileBox/FileBox.js:48]: can you please take out dumb data in a separate files. 
try to keep component logic and design separately (if it is necessary, then use custom hooks)

File: src/pages/Landing/components/Body/FileBox/FileBox.module.scss [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+        margin-bottom: 1rem;
+    }
+
+    > *:not(:last-child) {
+        margin-bottom: 1rem;
+    }
+
+    .Heading {
+        margin-bottom: 1rem;
+        font-size: 16px;
```
Comment (fargusmd) [inline src/pages/Landing/components/Body/FileBox/FileBox.module.scss:25]: rems pls

File: src/pages/Landing/components/Body/FileBox/FileBox.module.scss [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        color: $primary-color;
+    }
+}
+
+.Line {
+    width: 100%;
+    height: 2px;
```
Comment (fargusmd) [inline src/pages/Landing/components/Body/FileBox/FileBox.module.scss:36]: rems pls

File: src/pages/Landing/components/NavBar/NavBar.js [hunk 9]
```diff
@@ -1,32 +1,47 @@
...
           <img src={arrowDown} alt="Courses" />
         </li>
+        <li className={styles.NavItem}>Schedule</li>
+        <li className={styles.NavItem}>Study Groups</li>
+        <li className={styles.NavItem}>Ticket </li>
         <li className={styles.NavItem}>
-          Subjects
-          <img src={arrowDown} alt="Subjects" />
+          Profile
+          <img src={arrowDown} alt="Profile" />
```
Comment (fargusmd) [inline src/pages/Landing/components/NavBar/NavBar.js:30]: here we should implement svg icons

File: src/pages/Landing/components/NavBar/NavBar.module.scss [hunk 10]
```diff
@@ -1,53 +1,71 @@
...
+    &Active {
+      position: absolute;
+      width: 11px;
+      height: 11px;
+      background-color: #2da231;
+      border-radius: 50%;
+      top: 0;
+      right: 0;
+    }
+
```
Comment (fargusmd) [inline src/pages/Landing/components/NavBar/NavBar.module.scss:64]: rems pls

**Review by bragaru-i — CHANGES_REQUESTED** [review]
well done.
need to do some small fixes.
also try to implement Icon  component for svg class.
you added new plugin for react . well, can you inform team how it will work, and also have a look on import linter errors (disabling and not crushing intellisense from IDE)
ps: I added today proptypes wich will be a must for project( well, team is big, and proptypes will save some time from bugs typo)

**Comment by fargusmd** [comment]
also, in some places you added user-select: none.. well, maye someone wants to do some copy-paste from chats.. so not a ... good move

File: src/pages/Landing/components/Body/ChatBox/Input/Input.js (hunk 3 above)
Comment (truongduchuy) [reply src/pages/Landing/components/Body/ChatBox/Input/Input.js:14]: Which component needs to reusable? please clarify it

File: src/pages/Landing/components/Body/ChatBox/Input/Input.module.scss (hunk 4 above)
Comment (truongduchuy) [reply src/pages/Landing/components/Body/ChatBox/Input/Input.module.scss:14]: Which component needs to reusable? please clarify it

**Comment by truongduchuy** [comment]
this move to another PR

**Comment by fuelupeducation** [comment]
Need to change the route for study group common for teacher and student

**Pull request closed without merging**